assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -34,6 +36,26 @@ import { useConfig } from '../config';
 import Context from './context';
 import { flatternFormData, removeImagesFromPageTemplates } from './utils';
 
+// Important: Keep in sync with REST API preloading definition.
+const STORY_FIELDS = [
+  'id',
+  'title',
+  'status',
+  'slug',
+  'date',
+  'modified',
+  'excerpt',
+  'link',
+  'story_data',
+  'preview_link',
+  'edit_link',
+  'embed_post_link',
+  'publisher_logo_url',
+  'permalink_template',
+  'style_presets',
+  'password',
+].join(',');
```

Comment:
Do these values be stored in PHP and passed in via config object. Meaning keeping these fields in sync would be much easier.

Reply:
While keeping them in sync would be easier since there would be only 1 definition, I see it as more useful to have this definition closer to the actual source code where these fields are used. Moving them to PHP would be too far away from the JS code here.